wom/src/main/scala/wom/graph/expression/ExpressionNode.scala

```diff
@@ -58,12 +60,16 @@ object ExpressionNode {
                                                (identifier: WomIdentifier, expression: WomExpression, inputMapping: Map[String, OutputPort]): ErrorOr[E] = {
     val graphNodeSetter = new GraphNode.GraphNodeSetter[ExpressionNode]()
 
-    (for {
+    val builtExpressionNode = for {
       combined <- linkWithInputs(graphNodeSetter, expression, inputMapping)
       (evaluatedType, inputPorts) = combined
       expressionNode = constructor(identifier, expression, evaluatedType, inputPorts)
       _ = graphNodeSetter._graphNode = expressionNode
-    } yield expressionNode).leftMap(es => es.map(e => s"Cannot build expression for '${identifier.fullyQualifiedName.value} = ${expression.sourceString}': $e"))
+    } yield expressionNode
+
+    def safeSourceString(e: WomExpression) = Try(expression.sourceString).getOrElse("<<ERROR>>")
```

Comment:
`<<ERROR>>` seems a bit unspecific, would
```suggestion
    def safeSourceString(e: WomExpression) = Try(expression.sourceString).getOrElse("<<expected an expression, none found>>")
```
be an accurate and helpful thing to say?